app/views/templates/power_action.erb

```diff
@@ -0,0 +1,22 @@
+<%#
+kind: job_template
+name: Power Action - SSH Default
+job_category: Power
+description_format: '%{action} host'
+provider_type: SSH
+template_inputs:
+- name: action
+  description: Action to perform on the service
+  input_type: user
+  options: "restart\nshutdown"
+  required: true
+%>
+
+echo <%= input('action') %> host && sleep 3
```

Comment:
To give it a time to get the last output from the connection: we need the `echo` message to be received by the proxy (so that we know the disconnection is expected) before we loose the connection (and we can say that the disconnection error was expected, and not fail the whole script)